src/controls/MMapZoomControl/index.ts

```diff
@@ -192,6 +192,7 @@ class MMapZoomControl extends mappable.MMapComplexEntity<MMapZoomControlProps, D
     }
 
     protected override _onDetach(): void {
+        this._control.removeChild(this._zoom);
```

Comment:
so you can delete only `this._control`, its subtree will be automatically detached